src/crewai/knowledge/embedder/ollama.py

```diff
@@ -0,0 +1,82 @@
+import os
```

Comment:
i'd drop the ollama version. support openai, then let anyone bring their own embedder function (super easy) then have the knowledge_config setup like embedder_config setup for our rag storage